Src/Base/AMReX_GpuAtomic.H

```diff
@@ -152,18 +152,6 @@ namespace detail {
 #endif
     }
 
-#if defined(AMREX_USE_HIP)
```

Comment:
Do we already want to remove this? So far it's only deprecated.

Did they provide an equally fast replacement yet via `atomicAdd?`

Reply:
Doesn't seem so.  That's why it's marked "do-not-merge".  4.1 is not formally released yet.

Reply:
Oh got it, sorry I overlooked the label :see_no_evil:

Reply:
ROCm 5.0 has fixed the performance issue.